contrib/docker/docker-compose.yml

```diff
@@ -31,11 +35,12 @@ services:
       POSTGRES_PORT: 5432
       REDIS_HOST: redis
       REDIS_PORT: 6379
+      SUPERSET_PORT:  # Set port in .env file
       # If using production, comment development volume below
       #SUPERSET_ENV: production
       SUPERSET_ENV: development
     ports:
-      - 8088:8088
+      - ${SUPERSET_PORT}:${SUPERSET_PORT}
```

Comment:
@oliviermichaelis you don't have to use the same port for in-container process and host system: I think it would be a better idea to have a fixed port in the Dockerfile as before and if the user wants to change the port, it can change it here on left side of this line. See https://docs.docker.com/compose/compose-file/#ports

Reply:
@victornoel you are right, I wasn't thinking about the possibility to map a different host port to the container. Thanks for the feedback! :)